### ISSUE TITLE
Date component is placed in the dialog, part of the calendar is hidden.

### ISSUE THREAD

**Issue by oe-trader** [kintone/kintone-ui-component#250]
Put a date component in the dialog hides part of the calendar.

![コメント 2019-09-13 112752](https://user-images.githubusercontent.com/51394769/64833595-8e36e700-d619-11e9-8b42-df136bea1866.png)

I use a hack like this:

```
.kuc-dialog-body .date-time-container{
    position: absolute;
}
```

![コメント 2019-09-13 113016](https://user-images.githubusercontent.com/51394769/64833699-e1a93500-d619-11e9-863f-fb3ad2b8952f.png)

**Comment by youtr-s** [comment]
I'm sorry to late reply.
This problem has been solved in current version.
Would you please confirm it?

**Comment by oe-trader** [comment]
I've now confirmed the fix in the latest version (v0.7.2).
Thank you for your response!
